### DISCUSSION TITLE
ScriptReference/Graphics.DrawMeshInstancedProcedural

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#66] [Notes]
# ScriptReference/Graphics.DrawMeshInstancedProcedural

https://docs.unity3d.com/ScriptReference/Graphics.DrawMeshInstancedProcedural.html

**Comment by vertxxyz** [comment]
When drawing into the scene view it's important to do this at the rate the scene view renders using something like [`SceneView.beforeSceneGui`](https://docs.unity3d.com/ScriptReference/SceneView-beforeSceneGui.html). If you're enqueuing these draws from Update you may be outmatching the rendering of the scene view and can run out of the memory used to draw these objects.
